templates/templates/_further_reading_links.html

```diff
@@ -5,7 +5,13 @@
 {% endif %}
 
 <ul class='p-list'>
-{% for item in feed %}
+{% if feed is False %}
```

Comment:
It's not terribly important, but shouldn't this conditional be one line higher, so we don't end up with a `<div>` inside a useless `<ul>`? I tried it out in the inspector and it seems to display exactly the same.